### ISSUE TITLE
[Question]-- Can we attach a screenshot in TestLogger

### ISSUE THREAD

**Issue by saikrishna321** [anshooarora/extentreports#377]

**Comment by anshooarora** [comment]
Just like all other views, you can use any html in this view.

So you can do:

```java
extent.setTestRunnerOutput("<img src='./snapshots/1.png' alt='Snapshot' title='Snapshot' />");
```

**Comment by saikrishna321** [comment]
@anshooarora thanks.. i did that but the TestLogger is not scrollable and can we have the images placed  horizontal.

From the image below you can see i have reduced the browser size to view the other images 

<img width="1440" alt="screen shot 2016-03-19 at 10 58 21 am" src="https://cloud.githubusercontent.com/assets/4045041/13896835/b2d4c328-edc1-11e5-8c7e-ed68c5b2ab5b.png">

**Comment by anshooarora** [comment]
This was a fix part of 2.40.2.  You can rerun your tests with this version and this issue will go away.

**Comment by saikrishna321** [comment]
Sure ... How to place the images horizontally ?
